keps/prod-readiness/sig-network/1672.yaml

```diff
@@ -3,3 +3,5 @@ alpha:
   approver: "@wojtek-t"
 beta:
   approver: "@wojtek-t"
+stable:
+  approver: "@wojtek-t"
```

Comment:
@wojtek-t @marseel looking for a sign off here.

A few months back, myself and @marseel ran the Kubernetes performance tests to validate that this change did not regress performance significantly. IIRC, the tests included these two changes:
- https://github.com/kubernetes/perf-tests/pull/1986 
- https://github.com/kubernetes/perf-tests/pull/1987

I don't think we saw any noticeable regression in performance, but please correct me if I'm mis-remebering.

Reply:
yes, you are correct, we haven't seen noticeable regression.

Reply:
Yup - I remember that and I'm fine with this from scalability POV.
Some other comment below though :)